spec/factories/rws_ichiba_items.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :rws_ichiba_item, class: 'RakutenWebService::Ichiba::Item' do
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

<!-- Comment by [SideCI](https://sideci.com) -->